python-package/lightgbm/basic.py

```diff
@@ -4081,7 +4081,8 @@ def update(
         Returns
         -------
         is_finished : bool
-            Whether the update was successfully finished.
+            Whether there were no possible splits meeting the splitting criteria, that
```

Comment:
For consistency, we should also update the docs in l.4149.

Reply:
^ I agree with this (good catch!), any changes made to this should apply similarly to `__boost()`.